packages/core/src/typings/overrides.d.ts

```diff
@@ -1,18 +1,49 @@
 export const SECTIONS = {
-    ProductDetails: { components: ["Price"] },
-    ProductShelf: { components: ["ProductCard"] }
-  } as const;
-  
-  // export type ComponentOrProps =
-  //   | { Component: React.ElementType }
-  //   | { props: Record<string, unknown> };
-  
-  export type SectionOverride = {
-    [K in keyof typeof SECTIONS]: {
-      name: K;
-      components: {
-        [ComponentKey in typeof SECTIONS[K]["components"][number]]?: React.ElementType;
-      };
-    };
-  };
-  
\ No newline at end of file
+  Hero: {
+    components: ['Hero', 'HeroImage', 'HeroHeader'],
+  },
+  BannerText: {
+    components: ['BannerText'],
+  },
+  ProductDetails: {
+    components: [
+      'ProductTitle',
+      'DiscountBadge',
+      'BuyButton',
+      'SkuSelector',
+      'ShippingSimulation',
+    ],
+  },
+  ProductShelf: { components: ['ProductShelf', '__experimentalProductCard'] },
+  Navbar: {
+    components: [
+      'Navbar',
+      'NavbarLinks',
+      'NavbarLinksList',
+      'NavbarSlider',
+      'NavbarSliderHeader',
+      'NavbarSliderContent',
+      'NavbarSliderFooter',
+      'NavbarHeader',
+      'NavbarRow',
+      'NavbarButtons',
+      'IconButton',
+    ],
+  },
+  Breadcrumb: {
+    components: ['Breadcrumb', 'Icon'],
+  },
+} as const
+
+// export type ComponentOrProps =
```

Comment:
@icazevedo will this type be adjusted, or are going with `any`?

Reply:
It will be adjusted! There's a task for that!